chore: Update image source in authentication tutorial (#824)
MicrosoftDocs/fabric-docs: docs/workload-development-kit/authentication-tutorial.md

}
```

> :::image type="content" source="./media/authentication-tutorial/configure-workload-env-dev.png" alt-text="Screenshot showing configuration of local workload manifest xml file.":::

Reviewer: ```suggestion
:::image type="content" source="./media/authentication-tutorial/configure-workload-env-dev.png" alt-text="Screenshot that shows the configuration of a .env.dev file.":::
```
